### PR TITLE
Don't build JS/CSS

### PR DESCRIPTION
**Pull request by jkassemi**
Spike on removing JS/CSS from the build. This will be the first PR to the new content repository.

This generates HTML content that references JS/CSS from the current production www.vets.gov (by using the `appIncluder.js` middleware). We'll need separate build types that reference assets from each of the varying asset builds, but iterating toward that here.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,79 +1,16 @@
...
-    BUILDTYPE=${vars['buildtype']}
-    NODE_ENV=production
-    BRANCH_NAME=${env.BRANCH_NAME}
-    CHANGE_TARGET=${env.CHANGE_TARGET}
-    BUILD_ID=${env.BUILD_ID}
-    BUILD_NUMBER=${env.BUILD_NUMBER}
-    REF=${vars['ref']}
-  """.stripIndent()
-}
+def envNames = ['development', 'staging', 'production']
```
Comment (jkassemi) [inline Jenkinsfile]: I only configured one built type in the metalsmith config. We'll need the three.

File: content/pages/veteran-id-card/how-to-get.md [hunk 2]
```diff
@@ -1,124 +0,0 @@
----
```
Comment (jkassemi) [inline content/pages/veteran-id-card/how-to-get.md]: Removed content that was referenced in the ignore list. We should specify content availability in the targets.

File: script/build.js [hunk 3]
```diff
@@ -20,116 +18,29 @@ const permalinks = require('metalsmith-permalinks');
...
-const webpackDevServer = require('./metalsmith-webpack').webpackDevServerPlugin;
-const createSettings = require('../config/create-settings');
 const nonceTransformer = require('./metalsmith/nonceTransformer');
-const {
-  getRoutes,
-  getWebpackEntryPoints,
-  getAppManifests
-} = require('./manifest-helpers.js');
+const appIncluder = require('./metalsmith/appIncluder');
+const serve = require('metalsmith-serve');
```
Comment (jkassemi) [inline script/build.js]: didn't get this working to serve the local previews. Will need some love.

File: script/metalsmith/appIncluder.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const request = require('request');
```
Comment (jkassemi) [inline script/metalsmith/appIncluder.js]: needs ❤️. But not _too_ much for the first iteration. Let's just get the spike out and address issues as they pop up.

**Comment by ncksllvn** [comment]
Closing, just to clean up the repo as content editors migrate.

**Pull request closed without merging**